src/components/Dropdown/Dropdown.tsx

```diff
@@ -186,7 +238,7 @@ function useFocusContextInstance(
       setFocusedIndex(updatedFocusedIndex);
       setFocusedItemData(items[updatedFocusedIndex].itemData);
     }
-    setFocusedValue(updatedValue);
+    if (updateFocusedValue) setFocusedValue(updatedValue);
```

Comment:
what is the case where we would not want to update here?

Reply:
When toggling out of the dropdown, I check if the user has selected anything by comparing `value` and `focusedValue` - if they aren't the same that means the user's latest action was not selecting a filter. But in `updateFocusedItem`, `focusedValue` is always set to the same thing as `value` - the thing that usually is consistent with what is actually selected is `itemData`. So I added a that logic to make the checking a bit easier

Reply:
does that mean if the user types some text that exactly matches a filter value, even if they don't interact with the dropdown before clicking out, that filter would be applied?

Reply:
It would. I'm not sure how to remove that because the filter auto-focuses on a filter now, and afaik there'd be no way to check if there's any difference between a typed filter and a selected one except the text.

Reply:
would it be possible to just keep track with a boolean if the user has pressed some arrow key (or tab) and use that to determine whether to select a filter or not? it could reset whenever the user changes the input text

Reply:
chatted with Cindy, removing it since the new logic added in FilterSearch should take care of it